website/docs/plugin/framework/functions/testing.mdx

```diff
@@ -172,7 +172,7 @@ func TestEchoFunctionRun(t *testing.T) {
             request: function.RunRequest{
                 Arguments: function.NewArgumentsData(types.StringNull()),
```

Comment:
Unrelated to your original change, but I believe these `Arguments` fields also need to be updated in the example 🙃 
```suggestion
                Arguments: function.NewArgumentsData([]attr.Value{types.StringNull()}),
```